mozilla/contain-facebook
# Fixed #462 and #466 – GoFundMe Support and Negative Offset Badges

## Testing Steps

*GoFundMe Sign in support*
- Go to https://www.gofundme.com/sign-in
- Badge should be near login/sign up with Facebook buttons* 
  - *NOTE: The positioning may be off. This is a known issue due to the react/HTML position calculations.*

*GoFundMe Share support*
- Go to https://www.gofundme.com/f/mayte039s-college-tuition
- No badge should be in the top left corner
- Share buttons should be badged on hover. 

*Microsoft Share support*
- Go to https://docs.microsoft.com/en-us/windows/client-management/administrative-tools-in-windows-10
- No badge should be in the top left corner
- Share button should be visible after clicking "Share" 

## Screenshots

GoFundMe Login page: _(Note the offset)_
![image](https://user-images.githubusercontent.com/2692333/60903720-7a48c080-a237-11e9-8962-79b10efa05f9.png)

GoFundMe fund raiser page:
![image](https://user-images.githubusercontent.com/2692333/60903740-8896dc80-a237-11e9-989e-ff863d8abef9.png)

Microsoft Docs page:
![image](https://user-images.githubusercontent.com/2692333/60903707-6ef59500-a237-11e9-9161-739dfaeba643.png)
